### ISSUE TITLE
Suggestion for enhancement

### ISSUE THREAD

**Issue by DAG59** [gridcoin-community/Gridcoin-Research#466]
In the client comment messages section, would it be possible to broadcast of otherwise flash an upgrade notice for the leisure and mandatories?  This might be useful for those users who do not subscribe to the gridcoin forums or otherwise receive emails for the forums.   Once the upgrade for a client is complete the message can be dropped from the message marquee. ( flash in appropriate color to signify upgrade urgency. )

**Comment by Erkan-Yilmaz** [comment]
see: [important info to be displayed in the wallet (message of the day, ...)](https://github.com/gridcoin/Gridcoin-Research/issues/114)

**Comment by tomasbrod** [comment]
Yes this could be done easy using existing "message things". But will breake when forking. The user will not see messages from the other fork.

**Comment by tomasbrod** [comment]
Yes this could be done easy using existing "message things". But will breake when forking. The user will not see messages from the other fork.

**Comment by bentamircea** [comment]
What if the wallet could use it's peers to exchange the current block hash code and net weight (or whatever criteria is required) to validate it's state.

**Comment by tomasbrod** [comment]
The message could be loaded as soon as the transaction is received using normal tx propagation mechanism. Nodes still receive transaction even when on fork. The tx should have higher fees to propagate faster and further.

**Comment by denravonska** [comment]
Duplicate of #114.
